src/components/typography/typography.component.js

```diff
@@ -246,3 +246,4 @@ Typography.propTypes = {
 
 Typography.displayName = "Typography";
 export default Typography;
+Typography.displayName = "Typography";
```

Comment:
I'm sure you didn't mean to include the `displayName` twice! I assume a merge conflict incorrectly handled?